ISSUE_TEMPLATE.md

```diff
@@ -10,6 +10,8 @@ help and advice.
 ### Environment info
 Operating System:
 
+Installed version of CUDA and cuDNN (if any):
```

Comment:
Would it be worth asking for something like

ls -l /path/to/cuda/lib/libcud* ?

(It helps to find out where the symlinks are pointing to, permissions, etc).